src/components/DraggableBox.vue

```diff
@@ -76,37 +90,38 @@ const getImagePosition = () => {
     };
   }
 };
-
-onMounted(() => {
-  if (localStorage.getItem("ddQuizFormdata")) {
-    const storedData = localStorage.getItem("ddQuizFormdata");
-    if (storedData) {
-      const storeDDquizData: DDquizFormData = JSON.parse(
-        localStorage.getItem("ddQuizFormdata") ?? ""
-      );
-
-      const collectPositionFrLocal = storeDDquizData["collectPosition"];
-
-      snapItems.value = collectPositionFrLocal.map((item) => {
-        return { ...item, id: `snap${item.id}`, list: 2 };
-      });
-      items.value = collectPositionFrLocal.map((item, index) => {
-        return {
-          ...item,
-          list: 1,
-          position: {
-            x: 50,
-            y: index * 40 + 100,
-            width: item.dimensions.width,
-            height: item.dimensions.height,
-          },
-        };
-      }); //Todo: to fix a suitable position in Render Quiz Ticket
-
-      imageUrl.value = storeDDquizData.image;
-    }
-  }
-});
+const itemStyle = (ele: Item) => {
+  return getItemStyle(ele);
+};
+// This is the updating of reactive props received from main
+watch(
+  () => dragQuestion.value,
+  (newVal: OptionsDatabase[] | undefined) => {
+    let renderData;
+    newVal === undefined
+      ? (renderData = sampleDatabase)
+      : (renderData = newVal);
+    imageUrl.value = renderData[0].imgUrl; //TODO: Change Data Structur in Crucible(main)
+    snapItems.value = renderData.map((item, index) => {
+      return {
+        ...item,
+        id: `snap${index}`,
+        list: 2,
+        dimensions: { width: item.width, height: item.height },
+      };
+    });
+    items.value = renderData.map((item, index) => {
+      return {
+        ...item,
+        id: `${index}`,
+        list: 1,
+        dimensions: { width: 25, height: 25 },
+        position: { x: 50, y: index * 40 + 100 },
```

Comment:
Write comment above otherwise it looks like magic numbers.

Reply:
fixed